client/src/App.tsx

```diff
@@ -28,6 +29,7 @@ function App(): JSX.Element {
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/dashboard" component={Dashboard} />
+                <Route exact path="/profile/details" component={ProfileDetails} />
```

Comment:
Wouldn't we need to pass in the profile id of the user we are trying to see? Of course static for now, can still be passed in though.